packages/types/src/documents/document.ts

```diff
@@ -37,6 +37,13 @@ export enum DocumentType {
   USER_FLAG = "flag",
   AUTOMATION_METADATA = "meta_au",
   AUDIT_LOG = "al",
+  VIEW = "awd",
```

Comment:
Is this expected?

Reply:
woops - good catch, was just checking usage - thanks!